flixel/FlxCamera.hx

```diff
@@ -594,9 +594,9 @@ class FlxCamera extends FlxBasic
 	 * Internal variables, used in blit render mode to draw trianglesSprite on camera's buffer.
 	 * Added for less garbage creation.
 	 */
-	static var renderPoint:FlxPoint = FlxPoint.get();
+	static var renderPoint:FlxPoint = new FlxPoint();
 
-	static var renderRect:FlxRect = FlxRect.get();
+	static var renderRect:FlxRect = new FlxRect();
```

Comment:
why change this?

Reply:
Its incorrect usage, since its a static it is only ever gotten once. And never .put(). And if someone were to use .put on it it would break

Reply:
I disagree that it's "incorrect", it's makes absolutely no difference

Reply:
Oh i didnt know it was changed. It used to make a difference before the reworked in commit e721fca. Should i revert the commit?

Reply:
What difference did it make? I don't understand what you're referring to

Reply:
Eh just forget it. Do i need to do anything?